homeassistant/components/vevor_heater/sensor.py

```diff
@@ -0,0 +1,143 @@
+"""Support for Vevor BLE heater sensors."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfElectricPotential, UnitOfLength, UnitOfTemperature
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import VevorHeaterUpdateCoordinator
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS_MAPPINGS: dict[str, SensorEntityDescription] = {
+    "operational_mode": SensorEntityDescription(
+        key="operational_mode",
+        name="Operational Mode",
+        device_class=SensorDeviceClass.ENUM,
+    ),
+    "operational_status": SensorEntityDescription(
+        key="operational_status",
+        name="Operational Status",
+        device_class=SensorDeviceClass.ENUM,
+    ),
+    "input_voltage": SensorEntityDescription(
+        key="input_voltage",
```

Comment:
Please add entity_category so diagnostic entities don't appear on default dashboard

Reply:
Done - can you please double check the classification makes sense? The ones I labeled as "config" can technically be updated but the integration currently doesn't support it.

Reply:
Sensor entities should never be config since they can't actually interact with device beyond providing data. Diagnostic is likely better for this case